### PR TITLE
Support transpile

### PR DESCRIPTION
**Pull request by ohler55**
Support for borwsers that do not support ES6 was added using systemjs.

Note the extensions on the JavaScripts files are back to `.js` since we only have one source.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,4 +58,9 @@ view/pages/assets/js/wab.js
 node_modules
 package-lock.json
 
-export/assets
+export/assets/css
+export/assets/fonts
+export/assets/js/wab.js
+export/assets/js/ui.js
+export/assets/js/transpile.js
+
```
Comment (ashmaroli) [inline .gitignore]: why are you **not ignoring the entire `export/assets`** directory?

**Comment by ashmaroli** [comment]
What if we one needs to update the babel-plugins..? Isn't it hard now that there is no system in place..?

File: .gitignore (hunk 1 above)
Comment (ohler55) [reply .gitignore]: The systems and plugin-bagel are needed for the non-es6 to work. I haven't come up with a good way to gem them installed nicely yet. This gets things working until we can untie them. Do you have an alternative to offer?

**Comment by ohler55** [comment]
There is a note in the build-export script. We will have to implement something to to either have the user install the components or else install them in the export directory by doing a clone and copy files.

**Comment by ohler55** [comment]
What is the state where?

**Comment by ashmaroli** [comment]
> What is the state where?

Pardon me.. bt wha..??

**Comment by ohler55** [comment]
yes, :-)

**Comment by ashmaroli** [comment]
I'm not quite sure if you'd need to bundle all those `js` files  into wabur.. especially so in the case the `js.sourcemap` files..
I'm on the fence whether to go with this approach or compile ES6 into ES5 JS for every gem-release

**Comment by ohler55** [comment]
I would prefer to transpire ahead of time but I did not have success doing that with babel.

**Comment by ashmaroli** [comment]
hmm.. fine.. I'll approve if all this "external" code is moved into `export/assets/vendor` and the branch updated accordingly..

**Comment by ohler55** [comment]
Why `vendor` instead of `systems` and `plugin-babel` which are the repository names? With the JS in directories with the same names as the repos it makes it more clear where the code came from.

**Comment by ashmaroli** [comment]
its a Ruby convention (I think) to house "external" code in a `vendor` directory. so.. `export/assets/vendor/systemJS` and `export/assets/vendor/plugin-babel` etc..

**Comment by ohler55** [comment]
ok, sure. I'll also create an issue to figure out a better way of doing this.

File: .gitignore (hunk 1 above)
Comment (ashmaroli) [reply .gitignore]: any additional changes to be made here?

Comment (ohler55) [reply .gitignore]: No changed needed.

**Pull request merged**